setup_index.py

```diff
@@ -0,0 +1,16 @@
+#set up the index ONCE
+from llama_index.core import VectorStoreIndex, SimpleDirectoryReader, Settings
+from llama_index.embeddings.huggingface import HuggingFaceEmbedding
+
+
+
+Settings.embed_model = HuggingFaceEmbedding(
```

Comment:
you're doing this Settings.embed_model in three different places, maybe you should have a file that just does RAG settings?